app/graphql/mutations/eras/create_era.rb

```diff
@@ -0,0 +1,17 @@
+class Mutations::Eras::CreateEra < ::Mutations::BaseMutation
+  argument :user_id, ID, required: true
+  argument :name, String, required: false
+  argument :start_date, String, required: false
+  argument :end_date, String, required: false
+  argument :week_number, String, required: false
```

Comment:
Should this be an Int?

Reply:
Oh wait, we wouldn't ever accept week_number as an arg, right? It would only be available in the response (era/event type)

Reply:
Agreed! We don't want anyone to be able to manipulate the week number. They should change the date which will in turn change the week number. 

Also, I think Eras should have a start_week and and end_week.

Reply:
Done and done!